src/session-pool.ts

```diff
@@ -366,27 +377,7 @@ export class SessionPool extends EventEmitter implements SessionPoolInterface {
    * @type {number}
    */
   get available(): number {
-    const reads = this._inventory[types.ReadOnly];
-    const writes = this._inventory[types.ReadWrite];
-
-    return reads.length + writes.length;
-  }
-
-  /**
-   * Current fraction of write-prepared sessions in the pool.
-   * @type {number}
-   */
-  get currentWriteFraction(): number {
```

Comment:
This is a public method , removing this will also be a breaking change. We can mark this method as deprecated and probably return 0 ?

Reply:
done